downstream/assemblies/platform/assembly-aap-upgrading-platform.adoc

```diff
@@ -1,26 +1,17 @@
-
-
 ifdef::context[:parent-context: {context}]
 
+[id="aap-upgrading-platform"]
+= Upgrading to {PlatformName} {PlatformVers}
 
-
-[id="upgrading-platform"]
-= Upgrading your {PlatformName}
-
-
-:context: upgrading-platform
-
-
+:context: aap-upgrading-platform
 
 [role="_abstract"]
 
-To upgrade your {PlatformName} to the later version, download the desired version of the {PlatformNameShort} installer. Configure it with the same parameters you have used in previous installations, configure any new parameters, then run the installer.
-
-In some cases, you may need to configure other settings if you plan on upgrading other components of your {PlatformNameShort}, such as {ControllerName}. The following sections describes these steps in further detail.
+To perform an upgrade of your {PlatformName} to a later version, download the desired version of the {PlatformNameShort} installer. Configure the `inventory` file with the same parameters you have used in previous installations and add new parameters to enable any new features, then run the installer.
 
-include::platform/con-upgrade-planning.adoc[leveloffset=+1]
+include::platform/con-aap-upgrade-planning.adoc[leveloffset=+1]
 
-Once you have completed the necessary changes during upgrade planning, proceed by downloading the desired version of the {PlatformNameShort} installer.
+After you have completed the necessary upgrade planning, download the desired version of the {PlatformNameShort} installer.
```

Comment:
I'm not sure about putting sentences between include directives in an assembly like this. Perhaps you could combine this with the first sentence of the abstract.

To upgrade your {PlatformName}, start by reviewing planning information to ensure a successful upgrade.
You can then download the desired version of the {PlatformNameShort} installer, configure the `inventory` file in the installation bundle to reflect your environment, and then run the installer.